.changeset/happy-windows-drum.md

```diff
@@ -0,0 +1,5 @@
+---
+"@rocket.chat/meteor": patch
+---
+
+Stops the backend from saving invalid values for int settings (NaN for example)
```

Comment:
Fixed an issue that allowed `saveSettings` method to save `NaN` values on numeric settings.